examples/carplay-web-app/src/App.tsx

```diff
@@ -1,15 +1,11 @@
 import { useCallback, useEffect, useMemo, useState } from 'react'
 import { RotatingLines } from 'react-loader-spinner'
 import './App.css'
-import {
-  TouchAction,
-  findDevice,
-  requestDevice,
-  DongleConfig,
-} from 'node-carplay/dist/web'
+import { findDevice, requestDevice, DongleConfig } from 'node-carplay/dist/web'
```

Comment:
Was tired of seeing `dist/web` so opened https://github.com/rhysmorgan134/node-CarPlay/pull/44 to make it just `/web` so it looks more refined as a package